deploy-commands-function/go.mod

```diff
@@ -2,7 +2,11 @@ module github.com/PinkNoize/flavor-of-the-week/deploy-commands-function
 
 go 1.22.7
 
-require github.com/gorilla/websocket v1.4.2 // indirect
+require (
+	github.com/dimuska139/rawg-sdk-go v1.0.1 // indirect
+	github.com/gorilla/websocket v1.4.2 // indirect
+	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e // indirect
+)
```

Comment:
Remove the go.mod and go.sum changes unless you plan on adding changes to the go code in this PR.
`go mod tidy` should autodetect what is used and what isn't and autoremove it.